src/bin/ci/resources/papiv1_centaur.conf.ctmpl

```diff
@@ -24,6 +24,7 @@ backend {
         }
         filesystems {
           gcs.auth = "service_account"
+          gcs.project = "broad-dsde-cromwell-dev"
```

Comment:
TOL now that there are two properties in gcs it might look nicer as

```
filesystems {
  gcs {
    auth = "service_account"
    project = "broad-dsde-cromwell-dev"
  }
}
```